docs/getting-started-guides/ubuntu/upgrades.md

```diff
@@ -9,29 +9,53 @@ This page will outline how to manage and execute a Kubernetes upgrade.
 {% capture prerequisites %}
 This page assumes you have a working deployed cluster.
 
-## Assumptions
+**Warning:** You should always back up all your data before attempting an upgrade.
+Don't forget to include the workload inside your cluster!
+Refer to the [backup documentation](/docs/getting-started-guides/ubuntu/backups).
+{: .warning}
 
-You should always back up all your data before attempting an upgrade. Don't forget to include the workload inside your cluster! Refer to the [backup documentation](/docs/getting-started-guides/ubuntu/backups).
 {% endcapture %}
 
 {% capture steps %}
-## Preparing for an Upgrade
+## Patch kubernetes upgrades eg 1.7.0 -> 1.7.1
 
-See if upgrades are available. The Kubernetes charms are updated bi-monthly and mentioned in the Kubernetes release notes. Important operational considerations and change in behaviour will always be documented in the release notes. 
+Clusters are transparently upgrated to the latest Kubernetes patch release.
+To be clear, a cluster deployed using the 1.7/stable channel
+will transparently receive unattended upgrades for the 1.7.X Kubernetes
+releases.
+There is no disruption in the operation of clusters and the administration
```

Comment:
The upgrade causes no disruption to the operation of the cluster and requires no intervention from a cluster administrator.